tests/nfa/nfa.cc

```diff
@@ -2654,47 +2658,47 @@ TEST_CASE("Mata::Nfa::Nfa::unify_(initial/final)()") {
     }
 }
 
-TEST_CASE("Mata::Nfa::Nfa::get_epsilon_transitions()") {
+TEST_CASE("Mata::Nfa::Nfa::get_delta.epsilon_symbol_posts()") {
     Nfa aut{20};
     FILL_WITH_AUT_A(aut);
     aut.delta.add(0, EPSILON, 3);
     aut.delta.add(3, EPSILON, 3);
     aut.delta.add(3, EPSILON, 4);
 
-    auto state_eps_trans{ aut.get_epsilon_transitions(0) };
+    auto state_eps_trans{ aut.delta.epsilon_symbol_posts(0) };
```

Comment:
_symbol remove?

Reply:
Yes.

Reply:
Actually, this is not meant to be `epsilon_symbol` and `posts`, but `epsilon` and `sybol_posts`, as in, `SymbolPost` for `epsilon` from a given source. Therefore, the name should be kept as it is.